src/commands/interconvert-string-template-literal.ts

```diff
@@ -0,0 +1,51 @@
+import type { Command } from '../types'
+
+const PREFIX = /[\/:@]\s*/
+const STRING_LITERAL = /to-(?:string-literal|sl)|2sl/
+const TEMPLATE_LITERAL = /to-(?:template-literal|tl)|2tl/
+const NUMBER = /\s{0,}(.*)?/
+
+function getNodes<T>(nodes: T[], numbers: number[]) {
+  return numbers.length
+    ? numbers.map(n => nodes[n - 1]).filter(Boolean)
+    : nodes
+}
+
+export const interconvertStringTemplateLiteral: Command = {
+  name: 'interconvert-string-template-literal',
+  match: new RegExp(`^${PREFIX.source}(${STRING_LITERAL.source}|${TEMPLATE_LITERAL.source})${NUMBER.source}$`),
```

Comment:
I would prefer to always use static RegExp whenever possible. Dynamic regex is not good for JS engine optimization. We could use a comment to indicate the section.